docs/Azure_IoT_Hub_integration/Concepts/LwM2M_mappings.md

```diff
@@ -0,0 +1,61 @@
+# LwM2M mappings
+
+In this section you'll get to know how the mappings are arranged between the LwM2M protocol as used in Coiote DM and the data retrieval and processing mechanisms of the Azure IoT Hub, such as Device Twins, Direct Method and message routing interface (Device-to-cloud messaging).
+
+## Introduction
+
+The LwM2M protocol data model is organized as a three-level tree that has the following structure:
+
+ - **object** (e.g. a 'temperature sensor')
+    - **object instance** (e.g. 'temperature sensor #1', 'temperature sensor #2' etc.)
+        - **resource** (e.g. 'current temperature value')
+
+In terms of operations that can be performed on an LwM2M Client, an LwM2M Server can READ all of the data model entities, and, depending on their characteristics, may also WRITE to some of them, and EXECUTE some of them. Additionally, an LwM2M Server can also OBSERVE selected resources.
+
+This division into readable, writable, executable and observable data model entities is the basis for the mapping of LwM2M resources (as interpreted by Coiote DM) into Azure IoT Hub data processing mechanisms.        
+
+## LwM2M readable and writable resources
+
+Within the Coiote DM - Azure IoT Hub integration, readable and writable resources are usually interpreted as part of Azure Device twin data structure.
```

Comment:
We need to add an example here. It is not obvious how that mapping looks like. 
Let's add a screenshot or embed the snippet with raw json, to show what is the structure. That this is a tree with nested resources. For example /3/1/1 = "airquality-0-Valpariso is mapped to 

```
{
  "deviceId": "airquality-0",
   ...
  "properties": {
    "desired": {
      "$metadata": {
        "$lastUpdated": "2021-03-12T08:58:09.7856549Z"
      },
      "$version": 1
    },
    "reported": {
      "lwm2m": {
        "1": {
          "0": {
            "0": {},
            "1": {
              "value": 90
            },
            "4": {},
            "6": {},
            "7": {},
            "8": {}
          }
        },
        "3": {
          "1": {
            "1": {
              "value": "airquality-0-Valparaiso"
            }
          }
        },
   ....
```